### PR TITLE
Fix instructions for changing the default name for the participant controller

### PR DESCRIPTION
**Pull request by omichel**
Addresses #18.

### CODE REVIEW

File: ORGANIZER.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ There is no need to include all branches.
 ### 3. Webots Files
 
 - Replace/add all the files needed for your Webots simulation at the root of the repository, notably the folders:
-   - [controllers](controllers): it's where your robot and supervisor controllers will go. In competitions, there is at least one robot controller that the participants will modify and one supervisor controller that will measure how well their controller performs.
+   - [controllers](controllers): it's where your robot and supervisor controllers will go. In competitions, there is at least one robot controller, typically named [participant](controller/participant), that the participants will modify and one supervisor controller that will measure how well their controller performs. If you want to change the name of the participant controller, you will have to adapt the [README.md](README.md) and [webots.yml](webots.yml#L8) files accordingly.
```
Comment (Jean-Eudes-le-retour) [inline ORGANIZER.md]: Small typo in link
```suggestion
   - [controllers](controllers): it's where your robot and supervisor controllers will go. In competitions, there is at least one robot controller, typically named [participant](controllers/participant), that the participants will modify and one supervisor controller that will measure how well their controller performs. If you want to change the name of the participant controller, you will have to adapt the [README.md](README.md) and [webots.yml](webots.yml#L8) files accordingly.
```

**Review by Jean-Eudes-le-retour — CHANGES_REQUESTED** [review]
It's clearer now, just a small typo left in one of the links.

**Comment by Jean-Eudes-le-retour** [comment]
I find the name "participant" for the controller a bit odd, should we change it to something like "participant_controller"?

**Comment by omichel** [comment]
> I find the name "participant" for the controller a bit odd, should we change it to something like "participant_controller"?

Since it is already in the controllers directory, I believe this is not needed.

**Review by Jean-Eudes-le-retour — APPROVED** [review]
All good, cheers!

**Pull request merged**